<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>362</width>
      <height>462</height>
     </rect>
    </property>
    <property name="title">
     <string>Selection Parameters</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>% dams (of cows)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QLineEdit" name="potomciNP"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>MALES - selected population</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QLineEdit" name="pb"/>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>% pripust (od vhlevljenih)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>% naƒçrtnih parjenj (od NB)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>% pozTest (od mladih)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QLineEdit" name="vhlevljeni"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>FEMALES</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="NoNB"/>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLineEdit" name="mladi"/>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>% mladi (od vhlevljenih)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>% vhlevljeni (od NP)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="bm"/>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="pt"/>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="telF"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>No Newborns</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>% female calves &gt; cows</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>% of NB &gt; female calves</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QLineEdit" name="pripust1"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>366</width>
      <height>228</height>
     </rect>
    </property>
    <property name="title">
     <string>Time parameters</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Lactations per cow</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kraveUpE"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Dams chosen after lac no:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bmOdbiraE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Lactations per dam</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="bmUpE"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Bulls in test</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cakE"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Natural Service Years in use</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pripustUpE"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Tested bulls Years in use</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="pbUpE"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>365</width>
      <height>152</height>
     </rect>
    </property>
    <property name="title">
     <string>Dosage Parameters</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Dosage per young bull</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mladiDozE"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pripustDozE"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Dosage per tested bull</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pozitivnoTestDozE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Dosage per natural ser. bull</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="telM">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>184</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>% of NB &gt; male calves</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>523</y>
      <width>184</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>% of male calves &gt; bulls</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bik12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>523</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>467</y>
      <width>336</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MALES - nonselected population</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>AlphaSim</string>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="BurnInYNE">
       <property name="text">
        <string>Perform burn-in</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="StBurnInGenE"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Plot Genetic Gain</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="NoSelGen"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>from generation           </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Perform selection:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="SelFromGen"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>to generation</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="SelToGen"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>No Selected Generation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="DoMagic">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>650</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DoMagic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NoSires">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NoDams">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AlphaSimDir">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>640</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AlphaSimDirShow">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>640</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputWritten">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
